src/cosmic_ray/modules.py

```diff
@@ -12,9 +12,27 @@ def find_modules(module_path):
 
     Returns: An iterable of paths Python modules (i.e. *py files).
     """
-    if module_path.is_file():
-        if module_path.suffix == '.py':
-            yield module_path
-    elif module_path.is_dir():
+    if module_path.is_dir():
```

Comment:
Why have you restructured this function? It seems like you've just moved things around without changing what it does.

Reply:
If this doesn't have any functional change, I'd like it changed back.

Reply:
Why two tests?
There is an 3rd cases (silently ignored) ? When I read this previous code, I wonder if symbolic is take into account (I had to read is_file() specification to know that is yes).

Reply:
Path's can be things besides files and directories, e.g. fifos or sockets. The way it's coded, we silently ignore those possibilities, but I'm comfortable with that because a) the docs say what to pass in and b) the generator behaves in a well-defined way if we get any of those other types.

If anything, we should probably just add an `else` that raises a `ValueError` when we receive neither a file nor a directory.